e2e-tests/tests/migration-backup-s3/11-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
We skip step 10?

Reply:
Agree with @tplavcic. The rest looks good.

Reply:
Brought it back.